scripts/bfb-install

```diff
@@ -196,6 +196,8 @@ push_boot_stream_via_local_rshim()
     clear_boot_fifo
     # Set SP2.BIT2=1
     ${BF_REG} $(basename ${rshim_node}) 0x13000c48.64 0x4 >/dev/null
+    # Set SWINT2.BIT2
+    ${BF_REG} $(basename ${rshim_node}) 0x13000318.64 0x4 >/dev/null
```

Comment:
We may want to add BF_REG to the global variable list line above:

    # Global variables used:
    #   $bfb, $cfg, $rootfs, $pv, $rshim_node, $sudo_prefix

However, I tend to agree it's a bit of work to update this list on an ongoing basis, and its usefulness may be limited. So we could remove just remove these two comment lines instead.

Reply:
done